.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+
+[vcbuild.bat]
+end_of_line = crlf
+
+[*.{md,markdown}]
+trim_trailing_whitespace = false
+
+[{lib,src,test}/**.js]
+indent_style = space
+indent_size = 2
+
+[src/{**.h,**.cc}]
```

Comment:
FYI it would be slightly more idiomatic to write `[src/**.{h,cc}]`.  This will also work on more editors because early versions of EditorConfig did not support glob expansion inside `{...}` so asterisks only worked outside braces (this works properly in all EditorConfig libraries now though).